src/index.css

```diff
@@ -10,13 +10,42 @@ form .field-aligner {
 	max-width: 80%;
 }
 
-#root, #App {
+#App {
   height: 100%;
   display: flex;
   flex-direction: column;
   justify-content: space-between;
 }
 
+#root {
+  height:  100%;
+}
+
+#HashRouter {
+  flex: 1 0;
+}
+
+#HomePage {
+  background-image: url(./images/splash.svg);
```

Comment:
Our footer color is #00b6ad, but the decorative arrow color is #01aaa0. I think it may look better if they match.

Reply:
Also, when it's far from the edge, the shape looks a bit strange.

Reply:
Thanks for the color definition!  For some reason, my eyedropper wasn't quite doing the right thing @_@